agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -948,6 +969,9 @@ private static void setExtraAttributes(
               && !stringKey.startsWith("http.response.header.")) {
             return;
           }
+          if (stringKey.equals("isPreAggregated")) {
```

Comment:
let's name these new keys similar to our others, e.g. `applicationinsights.internal.is_pre_aggregated`, then they will be automatically filtered out